ast/fuzz_test.go

```diff
@@ -13,8 +13,10 @@ func FuzzParseStatementsAndCompileModules(f *testing.F) {
 	f.Fuzz(func(t *testing.T, input string) {
 		t.Parallel() // seed corpus tests can run in parallel
 		_, _, err := ParseStatements("", input)
-		if err != nil {
-			CompileModules(map[string]string{"": input})
+		if err == nil {
```

Comment:
Alright that one wasn't caught by the linter, but rather the linter complained about `CompileModules()` not having its error checked. And, well, reading this, the test didn't really make sense as written before 😳